### PR TITLE
Scotch + A List Apart

### PR DESCRIPTION
**Pull request by kenzable**

### CODE REVIEW

**Comment by masungwon** [comment]
Looks good to me!

File: a-list-apart/a-list-out.csv [hunk 1]
```diff
@@ -0,0 +1,488 @@
+title,author,tags,link,source
```
Comment (glebec) [inline a-list-apart/a-list-out.csv:1]: Again, if this CSV is produced via script, maybe can be `gitignore`d. In fact you might want to put all these CSVs into a `csvs` folder which is `gitignore`d to make things easier. @kenzable @masungwon

File: scotch-io/scotch-blog-out.csv [hunk 2]
```diff
@@ -0,0 +1,109 @@
+title,author,tags,link,source
```
Comment (glebec) [inline scotch-io/scotch-blog-out.csv:1]: Same comment.

File: a-list-apart/a-list-out.csv (hunk 1 above)
Comment (glebec) [reply a-list-apart/a-list-out.csv:1]: EDIT: unless the intent is to have some boilerplate example code as part of your repo, in which case you can make a case for including some pre-made CSVs. Depends on what exactly you intend to do with this data.

**Pull request merged**